### ISSUE TITLE
Build error on Linux

### ISSUE THREAD

**Issue by a1300** [AschPlatform/asch-smartdb-dist#1]
## Reproduction
```bash
# create new empty project
npm init --yes

# install asch-smartdb-dist package
npm install --save https://github.com/AschPlatform/asch-smartdb-dist

```
### Log
```log
npm ERR! Failed at the better-sqlite3@4.1.0 install script.
```

### Environment
OS: Linux Kubuntu 4.18.0-13-generic
node: v8.13.0

## Solution
Please use the __better-sqlite3__ version `4.1.3`!
```diff
-    "better-sqlite3": "=4.1.0",
+    "better-sqlite3": "=4.1.3",
```

Please have a look at https://github.com/JoshuaWise/better-sqlite3/issues/130
With the newest version of __better-sqlite3__ the issue is resolved!

**Comment by a1300** [comment]
This issue affects especially the __master__ branch of [ASCH](https://github.com/aschplatform/asch/tree/master) which is currently __unstable__ and very bad for new developers!

@sqfasd @liangpeili 
@AschPlatform/backend @AschPlatform/dapp  @AschPlatform/pm-dev

**Comment by eagleHovering** [comment]
@a1300 Thank you for your contribution，we will fix it in v1.5
